src/SFA.DAS.LevyTransferMatching.Api/Authentication/PolicyNames.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.LevyTransferMatching.Api.Authentication
+{
+    public static class PolicyNames
+    {
+        public const string PledgeAccount = "PledgeAccount";
```

Comment:
I'd be tempted to rename the policy to something where the intent is more clearly expressed, maybe "ManagePledge" or something like that. Likewise the requirement. "PledgeAccount" is just a bit non-obvious in what it _does_ I think?

Other than that though, looks great.